### DISCUSSION TITLE
Welcome to NHSRplotthedots Discussions!

### DISCUSSION THREAD

**Discussion by Lextuga007** [nhs-r-community/NHSRplotthedots#140] [Announcements]
I wanted to ask a general question of @tomjemmett about the task list he's used in pull requests and where that can be written in a package to help guide people when doing tasks and saw this feature in GitHub that I've not used before. It might be a really good place to have discussions about this package as those on NHS-R Slack will get lost eventually 😃 

Below is the information on discussions that pop up automatically.

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomjemmett** [comment]
for the task list, that comes from [.github/pull_request_template.md](https://github.com/nhs-r-community/NHSRplotthedots/blob/main/.github/pull_request_template.md), see [templates documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) for more 🙂

**Comment by ThomUK** [comment]
Good idea to store discussions longer-term. The slack messages become inaccessible surprisingly quickly! 👍
